src/main/java/seedu/duke/commands/ExchangeCommand.java

```diff
@@ -49,6 +50,9 @@ public void execute (Ui ui, AccountList accounts) {
             if (convertedAmount.compareTo(comparator) < 0) {
                 throw new ExchangeAmountTooSmallException();
             }
+            if (getNumberOfDecimalPlaces(amount) > 2) {
+                throw new AmountTooPreciseException();
+            }
```

Comment:
Thank you for adding this to exchange. I added max and min values to avoid implicit truncation from Java, but this should solve some precision issues I was having where 0.099 would round to 0.10 for example.